include/net/net_pkt.h

```diff
@@ -2399,6 +2399,15 @@ static inline int net_pkt_write_le32_new(struct net_pkt *pkt, u32_t data)
 	return net_pkt_write_new(pkt, &data_le32, sizeof(u32_t));
 }
 
+/**
+ * @brief Get the amount of data which can be read from current cursor position
+ *
+ * @param pkt Network packet
+ *
+ * @return Amount of data which can be read from current pkt cursor
+ */
+size_t net_pkt_data_to_read(struct net_pkt *pkt);
```

Comment:
What about calling this func "net_pkt_data_remaining()" ?

Reply:
yes, much better!